### ISSUE TITLE
What am i looking at?

### ISSUE THREAD

**Issue by 0hAlex** [NoRiskClient/issues#20]
### Your idea

![2023-11-19_15 22 19](https://github.com/NoRiskClient/issues/assets/85081415/a68a8474-00f5-404f-b3bd-0d42a48410b2)
Ziemlich low prio aber mod die einem eif zeigt was man anguckt ist ganz nice manchmal (bsp von Lunar, mod heißt da WAILA)

### Where would this be applied?

Client

### What platforms would this affect?

Not a launcher feature

**Comment by TimLohrer** [comment]
> Als Alternative sollte sonst auch [wthit](https://modrinth.com/mod/wthit) den Zweck erfüllen. Bzw brauchen wir dann überhaupt noch was eigenes?

Würde halt sagen es wäre cool wenn man das im gui toggeln könnte

**Comment by copyandexecute** [comment]
> Als Alternative sollte sonst auch [wthit](https://modrinth.com/mod/wthit) den Zweck erfüllen. Bzw brauchen wir dann überhaupt noch was eigenes?

Interessant. wir könnten eig echt bei paar mods die wir nicht selbst coden wollen einfach eine UI bereitstellen

**Comment by TimLohrer** [comment]
> > Als Alternative sollte sonst auch [wthit](https://modrinth.com/mod/wthit) den Zweck erfüllen. Bzw brauchen wir dann überhaupt noch was eigenes?
> 
> Interessant. wir könnten eig echt bei paar mods die wir nicht selbst coden wollen einfach eine UI bereitstellen 

True, und die dann featuren, damit die nicht per default immer installiert werden -> performance
